lang/promql/projects.txt

```diff
@@ -0,0 +1,3 @@
+# Git URLs of publicly-accessible projects to be used for parsing stats,
+# one per line.
+#
```

Comment:
You might have to create a new repo with standalone code snippets if that's not a common thing.

Reply:
I created `https://github.com/MichaHoffmann/semgrep-promql-mock-stats` and added it to that file!